src/psyclone/tests/dynamo0p3_builtins_test.py

```diff
@@ -295,56 +294,50 @@ def test_invalid_builtin_kernel():
             str(excinfo.value))
 
 
-def test_dynbuiltin_str():
+def test_dynbuiltin_str(dist_mem):
```

Comment:
You've been itching to do this for ages :-)

Reply:
It felt good :-)